lms/djangoapps/edxnotes/helpers.py

```diff
@@ -319,16 +325,23 @@ def get_notes(request, course, page=DEFAULT_PAGE, page_size=DEFAULT_PAGE_SIZE, t
     try:
         collection = json.loads(response.content)
     except ValueError:
+        log.error("Invalid response received from notes api: response_content=%s", response.content)
```

Comment:
Can we specify the exact reason in logged error which can tell us the problem. i.e. Incorrect/invalid JSON or Failed to load JSON.